docs/spec_draft.md

```diff
@@ -358,6 +359,56 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.concatenate
+
+### Semantics
+
+Concatenates sequence of tensors in `val` along dimension `dimension` and
+produces a tensor as a result. The tensors are of the same rank as the input
+operands except the specified dimension.
+
+### Operands
+
+| Name        | Type                                                                     |
+|-------------|--------------------------------------------------------------------------|
+| `dimension` | `i64`                                                                    |
```

Comment:
`si64`?

Reply:
Will fix to `si64` in the table, but keep `i64` in the examples, which is related to #22.